src/Inciter/AMR/active_element_store.h

```diff
@@ -0,0 +1,54 @@
+#ifndef AMR_active_element_store_h
+#define AMR_active_element_store_h
+
+#include <set>
+
+namespace AMR {
+
+    class active_element_store_t {
+        private:
+            std::set<size_t> active_elements;
+        public:
+            /**
+             * @brief Function to add active elements
+             *
+             * @param id The master_id this should map to
+             */
+            void add(size_t id)
+            {
+                // Check if that active element already exists
+                assert( !exists(id) );
```

Comment:
Please use `Assert` instead to make it consistent with the rest of the code (for better or worse). `Assert` is defined in [Base/Exception.h](https://github.com/quinoacomputing/quinoa/blob/master/src/Base/Exception.h#L36) and allows passing a message and will generate a stack trace in `DEBUG`.